.github/workflows/conformance.yaml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: CloudEvents
     strategy:
       matrix:
-        go-version: [1.17.x]
+        go-version: [1.18.x]
```

Comment:
nit: you can use the latest available go version in the workflows

Reply:
ok I changed them all to use "stable".

Reply:
I think one downside to this is that the first person to do a PR after a new release of golang might be forced to do things like run "gofmt" on everything if the format rules change. Just ran into that. But let's see how it goes...

Reply:
Yeah, not a big fan of these abstract version identifiers, especially when it comes to troubleshooting and you don't see immediately which version we are using in CI. Feel free to revert to 1.21 for now.

Reply:
ok back to 1.21